Code/Editor/IEditor.h

```diff
@@ -9,9 +9,9 @@
 #pragma once
 
 #ifdef PLUGIN_EXPORTS
-#define PLUGIN_API DLL_EXPORT
+#define PLUGIN_API AZ_DLL_EXPORT
```

Comment:
Don't we need to include `<AzCore/PlatformDef.h>` here to get the AZ_DLL_EXPORT/AZ_DLL_IMPORT defines "defined"?

Reply:
it must be defined somewhere else since I tried this in monolithic and non monolithic and it still worked across all plats... should I update it?  I wanted to change as little as possible.

Reply:
Well this is only working because some other cpp file is including PlatformDef.h before IEditor.h.

This can easily slightly break in non-unity builds, if PlatformDef.h happens to not be included in the individual source file

Reply:
I did compiel it in non-unity as well as unity mode to test.  But, there's no harm in adding it.  I'll add and revise.